### PR TITLE
Augmented data

### PR DESCRIPTION
**Pull request by vijay120**
This PR does the following:
1. Removes the test set so that all the limited data can be used for training and evaluation is done through cross-validation.
2. If the training set size is less than 10, the data is duplicated.
3. For the unrelated domain, negative training data is added.

### CODE REVIEW

File: tests/converter/test_df_converter.py [hunk 1]
```diff
@@ -93,5 +93,25 @@ def test_df_converter():
         "All right. So, you're transferring $200 from your checking to a savings. Is that right?"
     )
 
+    conv.process("hello!")
+    conv.assert_text(
```
Comment (minhtuev) [inline tests/converter/test_df_converter.py]: is it easier to assert domain/intent here?

Comment (vijay120) [reply tests/converter/test_df_converter.py]: I think its fine to assert the whole NLR since we want to test if the output behavior is consistent for a given DF app

File: mindmeld/converter/dialogflow.py [hunk 2]
```diff
@@ -259,13 +260,20 @@ def _create_intent_file(
...
+            if key in mindmeld_intent_directory:
+                with open(os.path.join(package_dir, default_intent_to_training_file[key])) as fp:
+                    for line in fp:
+                        all_text.append(line.strip())
 
-        target_test.write("\n".join(test))
-        target_train.write("\n".join(train))
+        # Double the size of the training set if there are less than 10 training examples.
+        # This is needed since the k-fold cross validation parameter is set to 10 for
+        # intent classification.
```
Comment (serapio) [inline mindmeld/converter/dialogflow.py]: We could reasonably set `k` to 3 or 5 for this converter. `k=10` is good for when the training set N>50 or so, but using the minimum k is better than duplicating data. If the data is still lower than k, then yeah we need to duplicate.

File: mindmeld/converter/dialogflow.py [hunk 3]
```diff
@@ -259,13 +260,20 @@ def _create_intent_file(
                 sentence += part
             all_text.append(sentence)
 
-        train, test = train_test_split(all_text, test_size=0.2)
+        for key in default_intent_to_training_file:
+            if key in mindmeld_intent_directory:
+                with open(os.path.join(package_dir, default_intent_to_training_file[key])) as fp:
+                    for line in fp:
+                        all_text.append(line.strip())
```
Comment (serapio) [inline mindmeld/converter/dialogflow.py]: Why do we combine welcome and unrelated data?

Comment (vijay120) [reply mindmeld/converter/dialogflow.py]: We are not combining the data from the two intents.

File: mindmeld/converter/dialogflow.py (hunk 2 above)
Comment (vijay120) [reply mindmeld/converter/dialogflow.py]: Understood, I will address this in a followup PR.
